.github/workflows/build.yml

```diff
@@ -127,15 +129,29 @@ jobs:
             static: true
             feature-set: none
             runs-on: windows-2022
+          - target: aarch64-apple-ios
+            args: "-A aarch64 --iphoneos --xnnpack --coreml -N"
+            static: true
+            feature-set: none
+            runs-on: macos-13
+          - target: aarch64-apple-ios-sim
+            args: "-A aarch64 --iphonesimulator --xnnpack --coreml -N"
+            static: true
+            feature-set: none
+            runs-on: macos-13
+          - target: aarch64-linux-android
+            args: "-A aarch64 --android --xnnpack --nnapi -N"
+            static: true
+            feature-set: none
+            runs-on: ubuntu-22.04
     steps:
       - name: Install cross-compile tools
-        if: matrix.target == 'aarch64-unknown-linux-gnu'
+        if: matrix.target == 'aarch64-unknown-linux-gnu' || matrix.target == 'aarch64-linux-android'
         run: |
           sudo apt update && sudo apt install -y gcc make gcc-11-aarch64-linux-gnu g++-11-aarch64-linux-gnu binutils-aarch64-linux-gnu
       - name: Remove unnecessary software
         if: runner.os == 'Linux'
         run: |
-          sudo rm -rf /usr/local/lib/android
```

Comment:
Let's add this back since CUDA builds are running out of space (apparently this folder is gigantic...)
```yml
- name: Remove unnecessary software (non-Android)
  if: runner.os == 'Linux' && matrix.target != 'aarch64-linux-android'
  run: |
    sudo rm -rf /usr/local/lib/android
```